staging/src/k8s.io/apiserver/pkg/util/openapi/enablement.go

```diff
@@ -68,7 +68,8 @@ func pruneEnums(schema *spec.Schema) *spec.Schema {
 				// note that the new lines before the header should be removed too,
 				// thus the slice range.
 				clone()
-				schema.Description = schema.Description[:headerIndex]
+				// stripping the header, taking account of two extra new lines before it.
+				schema.Description = schema.Description[:headerIndex-2]
```

Comment:
is it safe to assume the preceding lines are always blank? that seems fragile

Reply:
addressed in https://github.com/kubernetes/kubernetes/pull/114869 by trimming space rather than assuming two preceding newlines